shogun-config/pom.xml

```diff
@@ -72,6 +72,14 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <configuration>
+          <!-- we don't want jib to execute on this module since its a library -->
+          <skip>true</skip>
+        </configuration>
+      </plugin>
```

Comment:
The plugin will run only if it's explicitly configured for the module. So this block is not needed.

Reply:
JIB will run on each submodule defined in SHOGun's parent pom IMHO.
You can see this in output of `mvn clean compile jib:build` and in the plugins multi-module [example](https://github.com/GoogleContainerTools/jib/tree/master/examples/multi-module)

As far as I understand, (shared) libraries that are do not run as a standalone application need to be skipped from `jib:build` step. Maybe we need to introduce the plugin for reproducible builds as well,m what do you think?

Reply:
According to the logs from the last (stable) [build](https://github.com/terrestris/shogun/runs/1279433772?check_suite_focus=true) the execution will be applied to the desired modules only and I assume it's because the `jib` build process is bound to the `package`/`deploy` phases. So `mvn clean compile jib:build` is not necessarily needed / intended to be run.

Reply:
I've removed the unneeded config in 915aa882